build/com.liferay.ide-repository/pom.xml

```diff
@@ -39,7 +39,7 @@
                 <version>${tycho-version}</version>
                 <configuration>
                     <includeAllDependencies>false</includeAllDependencies>
-                    <finalName>liferay-ide-updatesite-${unqualifiedVersion}-b1-${maven.build.timestamp}</finalName>
+                    <finalName>liferay-ide-updatesite-${unqualifiedVersion}-rc1-${maven.build.timestamp}</finalName>
```

Comment:
hey @yanan  can you make this final name depend on a property from the build/parent/pom.xml?  THanks.